docker/config.deco.json

```diff
@@ -21,6 +21,14 @@
         "token": "{{ .instancerunner_token }}"
       }
     },
+    "databaseRunner": {
+      "type": "database",
+      "config": {
+        "endpointTemplate": "{{ .databaserunner_endpoint_tmpl }}",
+        "token": "{{ .databaserunner_token }}",
+        "auth_header": "X-Auth-Token"
```

Comment:
I don't think you can override this, it's the default

Reply:
Yeah, forgot I made it the default in the databaserunner, I was looking at the servicerunner where it's set in the config. Fixed.